packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -19,7 +19,7 @@ import { CollectionList } from './cards/list/CollectionList';
 import { MarketingCtaCard } from './marketingCta';
 import { MarketingCtaList } from './marketingCta/MarketingCtaList';
 import { FeedItemType } from './cards/common';
-import { PublicSquadEligibilityCard } from './squads/PublicSquadEligibilityCard';
+import { PublicSquadEligibilityCard } from './squads/cards/feed/PublicSquadEligibilityCard';
```

Comment:
I guess technically they just cards right (i see you split it into directory/) but eventually it's just a "squadCard" that's only shown on directory? (for now)

Maybe we should leave it as such, in case they eventually surface elsewhere. (obv non blocking/not important)

Reply:
I got this feeling SquadCards would be miss-interpreted for the cards in the squad feed which is not true so was thinking on how to clear this out. With context this is clear, but when new it's not, so not sure what do u think?

In the meantime, I think we're ok to move these into components/cards/squad rather then components/squad/cards?

Reply:
Hmm yeah see what you mean.
Just not sure on the `directory` addition?

To me:
`components/cards/squad` makes most sense?
Eventually we can maybe think about adding more readme data to storybook to describe what these actually are for? that should sort everything I think.

Reply:
Yeah true that could work, so at the moment then will remove the additional files add keep them 
flat within /cards/squad.